cmd/copyS2SMigrationS3Enumerator.go

```diff
@@ -229,7 +208,14 @@ func (e *copyS2SMigrationS3Enumerator) addTransferFromService(ctx context.Contex
 	// defines action need be fulfilled to enumerate bucket further
 	bucketAction := func(bucketInfo minio.BucketInfo) error {
 		// Note: Name resolving is only for destination, source's bucket name should be kept for include/exclude/wildcard.
-		resolvedBucketName, _ := r.ResolveName(bucketInfo.Name) // No error here, as already validated.
+		resolvedBucketName, err := r.ResolveName(bucketInfo.Name) // No error here, as already validated.
```

Comment:
This change isn't preserving all of the old behavior, and it probably should.   In particular, we sent messages to gclm in two cases in the old code: if the resolution failed, and if we change the name to resolve it.  THat glcm logging should be made to run here. (unless we have a good reason why we no longer want that logging, but I can't think of such a reason...)

Reply:
done